<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationSetupForm</class>
 <widget class="QWidget" name="SimulationSetupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_19">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="simulationTypeLabel">
         <property name="text">
          <string>Simulation Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="simulationType"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="settingsWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_18">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="generalGroup">
          <property name="title">
           <string>General</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="noOfStepsLabel">
               <property name="text">
                <string>Number of Steps</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="numberOfSteps">
               <property name="minimum">
                <number>-1</number>
               </property>
               <property name="maximum">
                <number>99999999</number>
               </property>
               <property name="value">
                <number>-1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="algorithmLabel">
               <property name="text">
                <string>Algorithm</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="algorithm"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="outputSettingsGroup">
          <property name="title">
           <string>Output</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="energyOutputFrequencyLabel">
               <property name="text">
                <string>Energy Output Frequency</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="energyOutputFrequency">
               <property name="maximum">
                <number>99999</number>
               </property>
               <property name="value">
                <number>500</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="positionOutputFrequencyLabel">
               <property name="text">
                <string>Position Output Frequency</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="positionOutputFrequency">
               <property name="maximum">
                <number>99999999</number>
               </property>
               <property name="value">
                <number>1000</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="velocityOutputFrequencyLabel">
               <property name="text">
                <string>Velocity Output Frequency</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="velocityOutputFrequency"/>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="forceOutputFrequencyLabel">
               <property name="text">
                <string>Force Output Frequency</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="forceOutputFrequency"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="minimisationGroup">
          <property name="title">
           <string>Minimisation Settings</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QLabel" name="minimisationMaximumForceLabel">
               <property name="text">
                <string>Maximum Force (kJ mol⁻¹ nm⁻¹)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="minimisationMaximumForce">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>9999999999.000000000000000</double>
               </property>
               <property name="value">
                <double>1000.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QLabel" name="minimisationStepSizeLabel">
               <property name="text">
                <string>Step Size (nm)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="minimisationStepSize">
               <property name="value">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="temperatureGroup">
          <property name="title">
           <string>Temperature</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="pressureGroup">
          <property name="title">
           <string>Pressure</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="electroStaticsGroup">
          <property name="title">
           <string>Electrostatic Potential</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QLabel" name="electrostaticAlgorithmLabel">
               <property name="text">
                <string>Long Range Electrostatics</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="electroStaticAlgorithm"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <widget class="QLabel" name="electrostaticCutoffRadiusLabel">
               <property name="text">
                <string>Cutoff Radius (nm)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="electrostaticCutoffRadius">
               <property name="maximum">
                <double>99999999.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <item>
              <widget class="QLabel" name="fourierSpacingLabel">
               <property name="text">
                <string>Fourier Spacing (nm)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="fourierSpacing">
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="value">
                <double>0.125000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="vanDerWaalsGroup">
          <property name="title">
           <string>Van der Waals Interaction</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <widget class="QLabel" name="vdwCutoffRadiusLabel">
               <property name="text">
                <string>Cutoff Radius (nm)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="vdwCutoffRadius">
               <property name="maximum">
                <double>99999999.000000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>116</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
